ikebuster/src/v2/peeking.rs

```diff
@@ -0,0 +1,176 @@
+        Transform::Integrity(IntegrityAlgorithm::HMAC_SHA2_512_256),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_2048),
+        Transform::KeyExchange(KeyExchangeMethod::ECP_Random_192),
+        Transform::KeyExchange(KeyExchangeMethod::ECP_Random_384),
+        Transform::KeyExchange(KeyExchangeMethod::ECP_Random_521),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_1024),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_3072),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_4096),
+        Transform::KeyExchange(KeyExchangeMethod::Curve_448),
+        Transform::KeyExchange(KeyExchangeMethod::Curve_25519),
+    ]);
+    let mut p2 = Proposal::new_empty(SecurityProtocol::InternetKeyExchange, Some(vec![]));
+    p2.add(vec![
+        Transform::Encryption(EncryptionAlgorithm::AES_CCM_8, Some(256)),
+        Transform::Encryption(EncryptionAlgorithm::AES_CCM_16, Some(256)),
+        Transform::Encryption(EncryptionAlgorithm::AES_GCM_8, Some(256)),
+        Transform::Encryption(EncryptionAlgorithm::AES_GCM_16, Some(256)),
+        Transform::Encryption(EncryptionAlgorithm::CHACHA20_POLY1305, None),
+        Transform::PseudoRandomFunction(PseudorandomFunction::HMAC_MD5),
+        Transform::PseudoRandomFunction(PseudorandomFunction::AES128_CMAC),
+        Transform::PseudoRandomFunction(PseudorandomFunction::HMAC_SHA2_256),
+        Transform::PseudoRandomFunction(PseudorandomFunction::HMAC_SHA2_384),
+        Transform::PseudoRandomFunction(PseudorandomFunction::HMAC_SHA2_512),
+        Transform::KeyExchange(KeyExchangeMethod::ECP_Random_192),
+        Transform::KeyExchange(KeyExchangeMethod::ECP_Random_384),
+        Transform::KeyExchange(KeyExchangeMethod::ECP_Random_521),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_1024),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_2048),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_3072),
+        Transform::KeyExchange(KeyExchangeMethod::MODP_4096),
+        Transform::KeyExchange(KeyExchangeMethod::Curve_448),
+        Transform::KeyExchange(KeyExchangeMethod::Curve_25519),
+    ]);
+    let proposals = vec![p1, p2];
+
+    if let Some((msg, _)) = make_new_hello_packet(proposals) {
+        let serialized_msg = msg.try_build().map_err(ScanError::GeneratorFailed)?;
+        let sent_bytes = socket
+            .send(serialized_msg.as_slice())
+            .await
+            .map_err(ScanError::Send)?;
+        debug!(
+            "Sent {} bytes with huge proposal to check availability",
+            sent_bytes
+        );
+
+        let mut accepted_indicators = 0;
+        let mut recv_buffer = [0u8; MAX_DATAGRAM_SIZE];
+        match socket.recv(&mut recv_buffer).await {
+            Ok(recv_bytes) => {
+                debug!(
+                    data = ?&recv_buffer[..recv_bytes],
+                    "Received {} bytes from responder",
+                    recv_bytes
+                );
```

Comment:
Split into two logging statements:

debug: received X bytes
trace: received this bytes: <BYTES>